src/pages/app/App.tsx

```diff
@@ -1,37 +1,39 @@
-import React from 'react';
+import React, { lazy, Suspense } from 'react';
 import {
   BrowserContextProvider,
   TranslationContextProvider,
 } from '@/pages/shared/lib/context';
 import { MessageContextProvider, WaitForStateLoad } from '@/app/lib/context';
 import browser from 'webextension-polyfill';
-import {
-  createHashRouter,
-  RouterProvider,
-  type RouteObject,
-} from 'react-router-dom';
+import { Route, Router, Switch } from 'wouter';
+import { useHashLocation } from 'wouter/use-hash-location';
 
 export const ROUTES = {
   DEFAULT: '/',
 } as const;
 
-const routes = [
-  {
-    path: ROUTES.DEFAULT,
-    lazy: () => import('./pages/PostInstall'),
-  },
-] satisfies RouteObject[];
+const Routes = () => {
+  return (
+    <Suspense>
+      <Switch>
+        <Route
+          path={ROUTES.DEFAULT}
+          component={lazy(() => import('./pages/PostInstall'))}
```

Comment:
Suspense is needed due to `lazy`. wouter doesn't native `lazy` like `react-router-dom`.